### DISCUSSION TITLE
Additional collection groups

### DISCUSSION THREAD

**Discussion by Rumcajsev** [outline/outline#4786] [Ideas]
Currently, all collections are stored "under" a single collapsible category. Additionally, users can add collections to starred ones, which is a separate collapsible category. 

It would be nice to be able to:
- define new collection categories 
- add collections to specific categories
- _potentially_ have additional level of access management - giving people access per collection category

Collections would then live under a specific category, with _Starred_ being a seperate category, defined per each user, which still groups collections from multiple categories.

This would make it easier to manage an instance with a lot of collections. An example below:
<img width="233" alt="Zrzut ekranu 2023-01-27 o 15 53 57" src="https://user-images.githubusercontent.com/23621302/215116538-5f8a6412-371e-49c0-b092-01bc8550906d.png">

**Comment by stereo9** [comment]
i was just thinking about the same.
for starters, i think it would already make a difference if it were possible to simply add a horizontal rule / `<hr>` in between to help structure navigation better

**Reply by stereo9** [reply]
@tommoor what do you think about this?
just realized how helpful this would be in terms of structuring as my team put tons of content on our outline the past few weeks

**Reply by tommoor** [reply]
I agree that another layer of organization is needed but I'm not sure the solution is necessarily this… we'll soon have "Shared documents" (or similar) in the sidebar here and I think it would be confusing to also allow user-generated headings mixed with the system ones
